src/components/content/order/common/utils/ScaleOrModifyWarnings.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+const ChangeFlavorServiceWillBeRestartedAndDataWillBeLost: string =
+    'Changing flavor will cause service to be restarted and data will be lost. Proceed?';
+const ChangeFlavorAndPropertyCauseDataWillBeLost: string = `Changing flavor and service property %s will cause service to be restarted and data to be lost. Proceed?`;
+const ChangePropertiesCauseDataWillBeLost: string = `Changing service properties %s will cause data to be lost. proceed?`;
+
+export enum CurrentOption {
+    SCALE = 'scale',
+    MODIFYPARAMETERS = 'Modify Parameters',
+}
+
+const findModifiedProperties = (originalData: Record<string, unknown>, currentData: Record<string, unknown>) => {
+    const modifiedFields = [];
+    for (const key in currentData) {
+        if (
+            currentData[key] !== '' &&
+            Object.prototype.hasOwnProperty.call(originalData, key) &&
+            currentData[key] !== originalData[key]
+        ) {
+            modifiedFields.push(key);
+        }
+    }
+
+    return modifiedFields;
+};
+
+const getModifiedPropertiesTip = (
```

Comment:
logic should be entirely based on the ModificationImpact flags. 
Check if property has changed. Check that specific property's modification impact flags. 
Check if flavor change is requested. Check that service's OCL to find out the impact of flavor change,